src/js/shared/components/Home.jsx

```diff
@@ -3,11 +3,25 @@ import Dropdown from './Dropdown';
 
 export default class Home extends Component {
 
+    para(source, winning, losing, candidates) {
```

Comment:
can we name this function something a bit more clear as to what it does? something like renderParagraph, e.g. Also, it would be cool to keep the parameter names the same as the ones that are being passed in, just for clarity